src/main/java/com/techiteasy/demo/controllers/ExceptionController.java

```diff
@@ -0,0 +1,30 @@
+package com.techiteasy.demo.controllers;
+
+import com.techiteasy.demo.exceptions.RecordNotFoundException;
+import com.techiteasy.demo.exceptions.TelevisionNameTooLongException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+//Deze klasse behandelt uitzonderingen (zoals de RecordNotFoundException)
+@Controller // Stap 5: voorzie deze klasse van de juiste annotatie
+public class ExceptionController {
+
+    // ExceptionHandler voor RecordNotFoundException
+    @ExceptionHandler(RecordNotFoundException.class)
+    public ResponseEntity<String> handleRecordNotFoundException(RecordNotFoundException ex) {
+        return ResponseEntity.status(404).body(ex.getMessage());
```

Comment:
Dit is helemaal goed. Als alternatief kun je in plaats van ".status(404)" ook ".notFound()" gebruiken met nog een message